README.md

```diff
@@ -10,9 +10,7 @@ The framework contains the majority of the logic for saving TCF-related data and
 
 ### Step 1: Include Our Framework
 
-- The package can be added Manually or using Cocoapods.
-
-Note: Just a heads up, the Swift Package Manager approach isn't ready for use just yet. We'll be rolling it out soon.
+- You can integrate the Framework into your application using one of the following methods:
 
 ### Manual Steps
```

Comment:
nit: maybe for the methods use four `#` instead of three since they are methods nested under Step 1? not super important tho

Reply:
Updated the readme with few more changes.